src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +115,12 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function setTemplateClass($templateClass)
+	{
+		if (!class_exists($templateClass) || !is_a($templateClass, UI\ITemplate::class, TRUE)) {
+			throw new \Nette\InvalidArgumentException('Template should be instance of ' . UI\ITemplate::class . ", $templateClass given.");
```

Comment:
* `Nette\InvalidArgumentException`
* "Class '$templateClass' does not implement ... ."